app/src/main/sentences/sl/calculator.dslf

```diff
@@ -0,0 +1,10 @@
+calculator: medium
```

Comment:
Parts of this file do not seem to have been translated, as I still see many english words